configure.in

```diff
@@ -84,44 +77,43 @@ if test "x$ac_cv_func_socket" = "xno"; then
 ]], [[
 socket(0, 0, 0);
 ]])],[ac_cv_func_socket=yes
-    AC_MSG_RESULT(yes)],[LIBS="$SAVELIBS"
-    AC_MSG_RESULT(no)])
+    AC_MSG_RESULT([yes])],[LIBS="$SAVELIBS"
+    AC_MSG_RESULT([no])])
 fi
 
-AC_MSG_CHECKING(whether to compile in debugging)
-AC_ARG_ENABLE(debug,
-  [AS_HELP_STRING([--disable-debug],
-    [do not compile in dynamic debugging system])],
-  [], enable_debug=yes)
+AC_MSG_CHECKING([whether to compile in debugging])
+AC_ARG_ENABLE([debug],
+  [AS_HELP_STRING([--disable-debug], [do not compile in dynamic debugging system])],
+  [], [enable_debug=yes])
 if test "$enable_debug" = "yes"; then
-   AC_DEFINE(ENABLE_DEBUGGING, 1,
-      [Define to compile in dynamic debugging system.])
+   AC_DEFINE([ENABLE_DEBUGGING], [1], [Define to compile in dynamic debugging system.])
 fi
-AC_MSG_RESULT($enable_debug)
+AC_MSG_RESULT([$enable_debug])
 
-AC_MSG_CHECKING(whether to use ISMAcryp code)
-AC_ARG_ENABLE(generic-aesicm,
-  [AS_HELP_STRING([--enable-generic-aesicm],
-    [compile in changes for ISMAcryp])],
-  [], enable_generic_aesicm=no)
+AC_MSG_CHECKING([whether to use ISMAcryp code])
```

Comment:
I don't find any current references to ISMAcryp in the code, there is some old documents. I have created a new issues #246 to follow up removing this if it is truly dead code after this patch is accepted/rejected.